statistics/handle/handle_test.go

```diff
@@ -619,7 +619,7 @@ func (s *testStatsSuite) TestCorrelation(c *C) {
 	result = testKit.MustQuery("show stats_histograms where Table_name = 't'").Sort()
 	c.Assert(len(result.Rows()), Equals, 2)
 	c.Assert(result.Rows()[0][9], Equals, "0")
-	c.Assert(result.Rows()[1][9], Equals, "0.828571")
+	c.Assert(result.Rows()[1][9], Equals, "0.8285714285714286")
```

Comment:
Here is changed because the old code use 

fmt.Sprintf( "%f"  ...) which truncate the data

and the new code use 

ParseWithParams("%?" ...)  for float64

I've checked the code and this change is reasonable, the old code must be a mistake.